src/components/Documentation/SidebarMenu/index.js

```diff
@@ -5,24 +5,37 @@ import scrollIntoView from 'dom-scroll-into-view'
 import PropTypes from 'prop-types'
 import includes from 'lodash.includes'
 
+import 'perfect-scrollbar/css/perfect-scrollbar.css'
+
 import DownloadButton from '../../DownloadButton'
 import LocalLink from '../../LocalLink'
 
-import { getParentsListFromPath } from '../../../utils/sidebar'
+import { getParentsListFromPath, getItemByPath } from '../../../utils/sidebar'
 
 import { OnlyDesktop } from '../../../styles'
 
 import { Menu, SectionLink, SectionLinks, Sections, SideFooter } from './styles'
 
-function SidebarMenuItem({ children, label, path, activePaths, onClick }) {
+function SidebarMenuItem({
+  children,
+  label,
+  source,
+  path,
+  activePaths,
+  onClick
+}) {
   const isActive = activePaths && includes(activePaths, path)
   const isRootParent =
     activePaths && activePaths.length > 1 && activePaths[0] === path
 
+  // ToDo: Dirty hack to make links with source=false work
```

Comment:
create a ticket or put it to some other visible place

Reply:
Done https://github.com/iterative/dvc.org/blob/5f589deee8fa32aca0f8c498947649fa02a87099/src/components/Documentation/SidebarMenu/index.js#L31